lib/vquic/ngtcp2.c

```diff
@@ -395,7 +409,117 @@ static int quic_init_ssl(struct quicsocket *qs)
   gnutls_server_name_set(qs->ssl, GNUTLS_NAME_DNS, hostname, strlen(hostname));
   return 0;
 }
+#elif defined(USE_WOLFSSL)
+
+static void wolfssl_logging(const int logLevel, const char *const logMessage)
+{
+    fprintf(stderr, " ** wolfssl[%d]: %s\n", logLevel, logMessage);
+}
+
+static WOLFSSL_CTX *quic_ssl_ctx(struct Curl_easy *data)
+{
+  struct connectdata *conn = data->conn;
+  WOLFSSL_CTX *ssl_ctx = wolfSSL_CTX_new(wolfTLSv1_3_client_method());
+
+#if 0
+  wolfSSL_SetLoggingCb(wolfssl_logging);
+  wolfSSL_Debugging_ON();
+#else
+  (void)wolfssl_logging;
+#endif
+
+  if(ngtcp2_crypto_wolfssl_configure_client_context(ssl_ctx) != 0) {
+    failf(data, "ngtcp2_crypto_wolfssl_configure_client_context failed");
+    return NULL;
+  }
+
+  wolfSSL_CTX_set_default_verify_paths(ssl_ctx);
+
+  if(wolfSSL_CTX_set_cipher_list(ssl_ctx, QUIC_CIPHERS) != 1) {
+    char error_buffer[256];
```

Comment:
`= { '\0' }`?

Reply:
`ERR_error_string_n()` really makes no assumptions about the content of the buffer passed. We could 0 it, if your prefer this, though.